harbor-cluster/templates/harborcluster.yaml

```diff
@@ -39,16 +40,19 @@ spec:
   # https://github.com/goharbor/harbor-operator/blob/master/docs/CRD/custom-resource-definition.md#standard-database
   
   database:
-    kind: PostgreSQL
+    kind: Zlando/PostgreSQL
     spec:
-      postgresql: 
-        username: postgres # Required
-        passwordRef: {{ .Values.postgresql.fullnameOverride }}
-        hosts:
-          - host: {{ .Values.postgresql.fullnameOverride }}
-            port: 5432  
-        sslMode: disable
-
+      zlandoPostgreSql:
+        operatorVersion: "1.7.0"
```

Comment:
does it install the operator ? what if the operator is already installed in the cluster ?

Reply:
no it doesn't install the operator. The operator should be installed beforehand. It is possible to enable its installation when installing harbor operator: https://github.com/goharbor/harbor-operator/blob/master/charts/harbor-operator/Chart.yaml
Important: when installing the zalando operator separatly be aware of this parameter: "secret_name_template". The default value given in zalando official chart is "{username}.{cluster}.credentials.{tprkind}.{tprgroup}" whereas harbor operator chart's defaullt value is "{username}.{cluster}.credentials"